analytics/analytics-ui/src/app/analytic/analytic.component.spec.ts

```diff
@@ -52,8 +52,12 @@ describe('AnalyticComponent', () => {
     component.model = {
       description: 'Test description',
       metaData: {
-        iconURL: 'Test url'
-      }
+        iconURL: '<svg xmlns=\'http://www.w3.org/2000/svg\' width=\'24\' height=\'24\' viewBox=\'0 0 24 24\'>' +
```

Comment:
The SVG icon should not be keyed as a URL since it isn't a URL.
The fields that are available in the metadata map should be documented.
You should consider using a specific class to contain this metadata - similar to the outputVisualisation field.

Reply:
Now uses icon instead of iconURL. Metadata class created.